dynamic-scoring-framework/samples/ai-workload-scorer/app/main.py

```
@@ -0,0 +1,349 @@
+            retrain_flags[node_label] = True
+
+        if retrain_flags[node_label]:
+            print(f"Training/Re-training model for {node_label}")
+            ts = TimeSeries.from_dataframe(
+                df_node, "timestamp", "total_power", fill_missing_dates=True, freq="60s"
+            )
+            model = LinearRegressionModel(lags=10)
+            try:
+                model.fit(ts)
+                model.save(model_path)
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)